ssc_utils/metric_switcher.py

```diff
@@ -187,14 +187,14 @@ def ad_impressions(self):
         )
 
         , device_data_impressions AS (
-            SELECT d.device_id,
-                   d.ds,
-                   d.platform_type,
-                   d.platform,
-                   d.device_first_seen_ts,
-                   d.first_exposure_ds,
-                   SUM(COALESCE(rev.ad_impression_total_count, 0))::float AS ad_impression_total_count,
-                   SUM(COALESCE(rev.gross_revenue, 0))::float AS gross_revenue
+            SELECT  d.device_id,
+                    d.ds,
+                    d.platform,
+                    d.platform_type,
+                    d.device_first_seen_ts,
+                    d.first_exposure_ds,
+                    SUM(rev.ad_impression_total_count) AS ad_impression_total_count,
```

Comment:
Why remove the coalesce and the cast to float? Did you test to make sure the results stay the same?

`tubidw.experiment_samplesize_input` has the same

Reply:
Good spot.